types/web3/eth/index.d.ts

```diff
@@ -85,11 +85,15 @@ export default interface Eth {
     currentProvider: Provider;
     estimateGas(tx: Tx, callback?: Callback<number>): Promise<number>;
     getAccounts(cb?: Callback<string[]>): Promise<string[]>;
+    getBalance(
+        address: string,
+        defaultBlock?: BlockType
+    ): Promise<BigNumber>;
     getBalance(
         address: string,
         defaultBlock?: BlockType,
-        cb?: Callback<number>
-    ): Promise<number>;
+        cb?: Callback<BigNumber>
```

Comment:
If `cb` parameter is left optional, it creates two conflicting method signatures:
`getBalance(address: string, defaultBlock?: BlockType): Promise<BigNumber>`
and
`getBalance(address: string, defaultBlock?: BlockType): void`

`cb` should be a required argument to make intent clear.

Reply:
Good catch